first class/first.js

```diff
@@ -0,0 +1,44 @@
+console.log("connected with external js file");
+
+var a=1;
+var b=2;
+if(a+b>4)
+{
+    console.log(a+b);
+}
+else if(a+b<1)
+{
+    console.log(a+b); 
+}
+else
+    console.log(a+b);
+
+for(var num=0;num<11;num=num+2)
```

Comment:
Try to write code as more readable, in real project it is more important to make it more readable. We uses short form in Competitive Programming but we will try to write it more expressively. Let's say, I would like to this lines as follows:

```javascript
for (var num = 0; num < 11; num = num + 2) {
  console.log(num);
}
```